src/components/EyeMap/style.jsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+const SEye = styled.main`
```

Comment:
Better to use another tag, main usually wrapping a lot of elements it could cause conflict here.